common-libs/user-management/src/UserManagementService.ts

```diff
@@ -201,7 +201,7 @@ export class UserManagementService {
       case CompleteLoginPasswordlessResult.ConfirmationCodeExpired:
         throw new SdkErrorFromCode('COR-17', { confirmationCode })
       case CompleteLoginPasswordlessResult.ConfirmationCodeWrong:
-        throw new SdkErrorFromCode('COR-5')
+        throw new SdkErrorFromCode('COR-5', { newToken: response.token })
```

Comment:
So are we returning new session or caching it?

Reply:
local in memory object do not work with cloud-wallet or any server side usage . so we return a session and client is responsible to send a correct session back . I add a proper section to a read.me of core library

Reply:
So we have to update contracts of cloud-wallet-api as well; and in-memory storage only remains for backwards compatibility with SDK consumers that are not aware of these changes, correct?